web/private/scripts/drush_post_deploy.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
-//Clear the cache to make sure database doesn't conflict
 echo "Start running drush deploy...\n";
 passthru('drush deploy -v -y 2>&1');
 echo "Done running drush deploy.\n";
+
```

Comment:
@kxwang 	I forgot to include this in my previous PR to automatically post the schema when deploying to Test and Live.

Reply:
Good catch! Does Sandbox also need this?

Reply:
I don't think so. Since it's a multidev, it should be handled by the corresponding code in the CircleCI script. If I understand correctly, CircleCI is used to build dev and the multidev sites (including sandbox), but Test and Live use these Quicksilver scripts.